sdk/src/claim.rs

```diff
@@ -3065,16 +3065,73 @@ impl Claim {
                                 continue;
                             }
                             Err(e) => {
-                                log_item!(
-                                    claim.assertion_uri(&hash_binding_assertion.label()),
-                                    format!("asset hash error, name: {name}, error: {e}"),
-                                    "verify_internal"
-                                )
-                                .validation_status(validation_status::ASSERTION_DATAHASH_MISMATCH)
-                                .failure(
-                                    validation_log,
-                                    Error::HashMismatch(format!("Asset hash failure: {e}")),
-                                )?;
+                                // If standard asset hard binding fails, try multi-asset hash validation.
+                                // Only one multi-asset hash assertion is allowed per manifest.
+                                if let Some(assertion) =
```

Comment:
Where is the test for multiple MultiAssetHash assertions.  The spec say there should be only one.